core/utils.py

```diff
@@ -32,6 +33,7 @@
 import time
 import unicodedata
 import urllib
+from urllib import request
```

Comment:
Do not use `from`, just use `import urllib` and then do `urllib.request`

Reply:
Yes, I did but I was getting this error
### core/utils.py:1282: error: Module has no attribute "request"  [attr-defined].

Reply:
It looks like someone updated import like this `import urllib.request`. I have also updated the code accordingly.